api/src/opentrons/protocol_engine/commands/blow_out.py

```diff
@@ -0,0 +1,81 @@
+"""Blow-out command request, result, and implementation models."""
+from __future__ import annotations
+from typing import TYPE_CHECKING, Optional, Type
+from typing_extensions import Literal
+from pydantic import BaseModel
+
+from .pipetting_common import BasePipettingParams
+from .command import AbstractCommandImpl, BaseCommand, BaseCommandCreate
+
+from opentrons.hardware_control import HardwareControlAPI
+
+
+if TYPE_CHECKING:
+    from ..execution import MovementHandler
+    from ..state import StateView
+
+BlowOutCommandType = Literal["blow-out"]
```

Comment:
Needs to match JSONv6

```suggestion
BlowOutCommandType = Literal["blowout"]
```